apollo-router-scaffold/templates/base/Cargo.toml

```diff
@@ -27,6 +27,7 @@ apollo-router = "1.4.0"
 {{/if}}
 async-trait = "0.1.52"
 futures = "0.3.21"
+linkme = "0.3.6"
```

Comment:
This can also be removed

Reply:
Yeah. I did think about that. I figured that leaving it in there won't cause an issue, but I can remove it to be consistent.

Reply:
done: https://github.com/apollographql/router/pull/2181/commits/57c557e1705707cd57c99097efa0dc13686817a0